sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleCounterSdk.java

```diff
@@ -73,13 +73,12 @@ public int hashCode() {
     return result;
   }
 
-  private static final class BoundInstrument extends AbstractBoundInstrument
-      implements BoundDoubleCounter {
+  static final class BoundInstrument extends AbstractBoundInstrument implements BoundDoubleCounter {
 
     private final boolean monotonic;
 
-    BoundInstrument(LabelSet labelSet, boolean monotonic) {
-      super(labelSet);
+    BoundInstrument(boolean monotonic) {
+      super(null);
```

Comment:
don't we need to be passing the labelset in to the ABI here?

Reply:
This is now the aggregation, will pass in a later PR :)